climada/hazard/tc_tracks_forecast.py

```diff
@@ -83,7 +87,8 @@ class TCForecast(TCTracks):
         data (list(xarray.Dataset)): Same as in parent class, adding the
             following attributes
                 - ensemble_member (int)
-                - is_ensemble (bool)
+                - is_ensemble (bool; if True, the simulation is a high resolution
```

Comment:
That name sounds quite obscure. Why would the word 'ensemble' = 'high resolution deterministic run'? I actually do not see why this would be a base class variable. Not all forecast models have the concept of 'high resolution deterministic run', which in itself is unclearly defined as it is unclear what 'high resolution' and 'deterministic' mean. What is a 'deterministic' run for a tropical cyclone track forecast model?

Reply:
`is_ensemble` should be `False` when the track is from the high-res deterministic model (see above suggestion). this is how it's implemented in the code.

this setup is true for a lot of meteorological products, where an agency provides _n_ ensemble model simulations (here 51, i think) plus their 'deterministic forecast', a higher resolution better model run as their single best guess. (the term 'deterministic' here is chosen to contrast with the 'probabilistic' ensemble members.) for something like the TCForecast it makes sense to think of this as a 52nd ensemble member. it should be distinguishable somehow in case e.g. you want to exclude it from an analysis, or you want to plot a representative track for the storm

Reply:
OK, thanks for the explanation. The name is however still quite cryptic ihmo. Maybe choose another name (unless it is some general standard). And could you maybe expand the docstring accordingly?